bootstrap/bootstrap.ps1

```diff
@@ -90,7 +90,7 @@ if ($env:buildtools_source -ne $null)
 }
 $packagesPath = Join-Path $RepositoryRoot "packages"
 $dotNetExe = Join-Path $cliLocalPath "dotnet.exe"
-$restoreArgs = "restore $projectJson --packages $packagesPath --source $buildToolsSource --source $nugetOrgSource"
+$restoreArgs = "restore $projectJson --packages $packagesPath --source $buildToolsSource --source $nugetOrgSource $($env:init_tools_restore_args)"
```

Comment:
I wouldn't change the name to BOOT_STRAP_RESTORE_ARGS, also I forget exactly where this is used but could we instead make this an optional parameter to both scrips instead of an environment variable?